test/models/report_test.rb

```diff
@@ -3,7 +3,51 @@
 require 'test_helper'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test 'editable?' do
+    alice_report = reports(:alice)
+    alice = users(:alice)
+    bob = users(:bob)
+
+    assert alice_report.editable?(alice)
+    assert_not alice_report.editable?(bob)
+  end
+
+  test 'created_on' do
+    alice_report = reports(:alice)
+
+    assert_equal Date.new(2024, 10, 9), alice_report.created_on
+  end
+
+  test 'save_mentions' do
+    alice = users(:alice)
+    bob_report = reports(:bob)
+    carol_report = reports(:carol)
+
+    alice_report = alice.reports.create(title: 'create_mention', content: <<~TEXT)
```

Comment:
```suggestion
    alice_report = alice.reports.create!(title: 'create_mention', content: <<~TEXT)
```

保存に失敗したらすぐ気づけるように ! 付きのcreate!を使うのがおすすめです。（updateも同様）